samples/servicecontrol/adapter-sqlserver/SCTransportAdapter_0/Adapter/Program.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Configuration.AdvanceExtensibility;
+using NServiceBus.Routing;
+using NServiceBus.Transport.SQLServer;
+using ServiceControl.TransportAdapter;
+
+class Program
+{
+    static void Main()
+    {
+        Console.Title = "Samples.ServiceControl.SqlServerTransportAdapter.Adapter";
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        #region AdapterTransport
+
+        var adapterConfig = new TransportAdapterConfig<SqlServerTransport, SqlServerTransport>("ServiceControl.SqlServer.Adapter");
+
+        #endregion
+
+#pragma warning disable 618
+
+        #region EndpointSideConfig
+
+        adapterConfig.CustomizeEndpointTransport(transport =>
+        {
+            transport.EnableLegacyMultiInstanceMode(Connections.GetConnection);
```

Comment:
It is required and this is the essence of the sample: it shows how to use SC in the multi-instance mode without hacks to SC configuration. I agree however that `EnableLegacyMultiInstanceMode` is a very bad name :(